lib/v2/metmuseum/exhibitions.js

```diff
@@ -0,0 +1,31 @@
+const got = require('@/utils/got');
+
+function generateExhibitionItem(result) {
+    return {
+        title: result.title,
+        link: `https://www.metmuseum.org${result.url}`,
+        description: result.description,
+        pubDate: result.startDate,
```

Comment:
```suggestion
        pubDate: parseDate(result.startDate),
```
docs: https://docs.rsshub.app/en/joinus/pub-date.html